distributed/scheduler.py

```diff
@@ -7920,6 +7920,62 @@ def adaptive_target(self, comm=None, target_duration=None):
             to_close = self.workers_to_close()
             return len(parent._workers_dv) - len(to_close)
 
+    def acquire_replicas(self, addr: str, keys: list, *, stimulus_id: str):
+        """Asynchronously request a worker to acquire a replica of the listed keys from
+        other workers. This is a fire-and-forget operation which offers no feedback for
+        success or failure, and is intended for housekeeping and not for computation.
+        """
+        parent: SchedulerState = cast(SchedulerState, self)
+        ws: WorkerState
+        ts: TaskState
+
+        who_has = {}
+        for key in keys:
+            ts = parent._tasks[key]
+            who_has[key] = {ws._address for ws in ts._who_has}
+
+        self.stream_comms[addr].send(
+            {
+                "op": "acquire-replicas",
+                "keys": keys,
+                "who_has": who_has,
+                "stimulus_id": stimulus_id,
+            },
+        )
+
+    def remove_replicas(self, addr: str, keys: list, *, stimulus_id: str):
```

Comment:
Same here. The scheduler immediately forgets the replicas exist, but the workers don't drop the data immediately before the function returns. `request_remove_replicas` or `request_worker_remove_replicas` perhaps? Again, no strong thoughts, up to you.